src/utils.ts

```diff
@@ -7,19 +8,15 @@ import * as React from 'react'
  */
 export function equals<T>(a: T, b: T): boolean {
   if (isImmutable(a) && isImmutable(b)) {
-    return a.is(b)
+    return is(a, b)
   }
   return a === b
 }
 
-export type Immutable = {
-  is(b: any): boolean
-}
-
-export function isImmutable(a: any): a is Immutable {
-  return !!a && typeof a === 'object' && (
-    '@@__IMMUTABLE_ITERABLE__@@' in a
-    || '@@__IMMUTABLE_RECORD__@@' in a
```

Comment:
Please keep this as-is. `a` is an `any`, not an `object`.

Reply:
OK!